PF/Assignment1/anshitaarya/prg4Test.java

```diff
@@ -0,0 +1,59 @@
+package anshitaarya;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class prg4Test {
+
+	prg4 assign=new prg4();
+	int a[]={1, 5, 9, 25};
+	int size[]={12,7,2,5};
+	int expected[][]={{1,1,0,1,12},{2,5,8,13,19},{3,9,11,20,21},{4,25,0,25,29}};
+	
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	 int isArrayEqual(int expected[][], int output[][])
```

Comment:
ACTION: Role of this function is to check whether two input arrays are equal or not. So this method should return a boolean(true,false) instead of an integer.